charts/csi-isilon/templates/controller.yaml

```diff
@@ -310,6 +296,16 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         {{- if not $encrypted }}
+        - name: csi-metadata-retriever
+          image: {{ required "Must provide the CSI metadata retriever container image." .Values.images.metadataretriever }}
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          command: [ "/csi-metadata-retriever" ]
+          env:
```

Comment:
I'm not a psacle expert, but would we need to add the missing args?
Previously these args were present
```          args:
            - "--csi-address={{ $driverSockPath }}"
            - "--timeout=120s"
            - "--v=5"
```
and csi retreiver endpoint was set to:
```
  - name: CSI_RETRIEVER_ENDPOINT
              value: /var/run/csi/csi_retriever{{ $csiSidecarSuffix }}.sock
```
instead of a hardcoded string

Reply:
Regarding args: I removed them, since no command line arguments are parsed by the container binary, all config is passed via env.
Regarding endpoint: this variable is not needed, it will always be empty.